### ISSUE TITLE
Support for AppManager to be able replace the built-in package installer front-end app

### ISSUE THREAD

**Issue by Tharow** [MuntashirAkon/AppManager#1404]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

For the App Manager to be able to take the place of the built-in front end for the Package Installer.

### Describe the solution you'd like

I will attempt to create a Universal Front-End for the Package Installer which then would be come the biases for the App Manager's Implementation of this feature 

### Describe alternatives you've considered

An App that would redirect all Package Management intents to App-Manager

### Additional context

I have a self imposed deadline of the end of this year for the completion of a universal front-end, required for this feature.

**Comment by Tharow** [comment]
> Request: installer not creating app shortcuts to start page (launcher). Add option to create shortcuts?

Not related, and most launchers already support turning that feature off.

**Comment by MuntashirAkon** [comment]
@Tharow: Any updates?

**Comment by Tharow** [comment]
> @Tharow: Any updates?

Yes, the biggest thing I'm trying to get working is loading sessions I have a way to accept session installation Confirmation and confirm it.

We'd need to add android.permission.PACKAGE_VERIFICATION_AGENT if we wanted to control incoming installs (it would let us intercept any install including those from markets) and modify them as we pleased.

Good news we don't have to touch apex as it's not handled by the front end, infact nothing besides the package server can deal with them at all and it just redirects it to the ApexManager. adb install was the only way I found to install them. On stock and lineageos as thoses are the only kind of environment this feature will work on. Due to the asop build requirement. Maybe magisk but I've never used it.

The other required configuration files have been made
For sysconfig, privapp-permission, and default-permission

Uninstallation is easy once a dialog is made for it.

**Comment by MuntashirAkon** [comment]
> But apps installed with App manager installer doesn't create shortcut to launcher/home/drawer despite launchers setting "auto add shortcuts for new apps" is enabled.
> 
> App managers installer is good with features but then one needs to manually add shortcuts :/

Unfortunately, our ability to fix issues is restricted to what we control. Since we cannot control your launcher, we cannot do anything about it. We expect that any Launcher3 (AOSP) based launcher will support automatic shortcut creation. We have no control over the behavior of your favorite launcher.

**Comment by Tharow** [comment]
First Successful Build!

![Image](https://github.com/user-attachments/assets/8ae6d3de-0b54-4717-a2b7-e77d85107633)
